src/Microsoft.AspNetCore.Authentication/AuthenticationBuilder.cs

```diff
@@ -97,6 +97,10 @@ public EnsureSignInScheme(IOptions<AuthenticationOptions> authOptions)
             public void PostConfigure(string name, TOptions options)
             {
                 options.SignInScheme = options.SignInScheme ?? _authOptions.DefaultSignInScheme;
+                if (options.SignInScheme == name)
```

Comment:
string.equals with a specific comparer (ordinal?)

Reply:
Hrm so that's actually an interesting question, we don't define the mixed casing behavior anywhere today.  Options just flows the name as is, we use constants mostly so this behavior actually isn't clearly defined today.  I guess we probably should be case insensitive for the purposes of auth schemes?

Reply:
I think it has always been explicitly case sensitive.